src/Easydropdown/Easydropdown.test.ts

```diff
@@ -6,11 +6,11 @@ import Renderer from '../Renderer/Renderer';
 
 import Easydropdown  from './Easydropdown';
 
-const createSelect = () => {
+const createSelect = (selectAttributes = '') => {
```

Comment:
let's make this an array, it will read better:

```ts
const createSelect = (selectAttributes: string[] = []) => {
   ...
   <select ${selectAttributes.join(' ')}>
   ...
}
```